tools/util/include/cutlass/util/reference/device/tensor_compare.h

```diff
@@ -95,16 +95,18 @@ __global__ void
 
   size_t idx = ThreadIdxX() + BlockDimX() * BlockIdxX();
 
-  for (; idx < capacity; idx += GridDimX() * BlockDimX()) {
-
+  //for (; idx < capacity; idx += GridDimX() * BlockDimX()) {
+  if (idx < capacity ){
```

Comment:
why this change?

Reply:
Hi @taozha2, do you have any comments here?